tests/batchnorm_test.py

```diff
@@ -0,0 +1,22 @@
+# pylint: disable=no-member, arguments-differ, missing-docstring, invalid-name
+import unittest
+import torch
+from e3nn.batchnorm import BatchNorm
+
+
+class Tests(unittest.TestCase):
+    def test_that_it_runs(self):
+        Rs = [(3, 1), (4, 3)]
+        m = BatchNorm(Rs)
+
+        m.to(dtype=torch.float64)
+        x = torch.randn(10, sum(mul * d for mul, d in Rs), dtype=torch.float64)
+        m(x)
+
+        m.to(dtype=torch.float32)
```

Comment:
Rather than just two examples, do we want to add 10 for test?
Additionally, should we test that on average, for inputs of varying variance (perhaps an additional outside look to the number of tests), that batch norm is able to bring the variance back to 1?

Reply:
You are right, it was laziness